products/hr-build/src/server.ts

```diff
@@ -13,7 +17,36 @@ const appSecret = process.env.SERVER_SECRET ?? 'secret'
 
 const dbUri = process.env.MONGODB_URI ?? 'mongodb://localhost:27017'
 
+const WORKSPACE = process.env.WORKSPACE ?? 'workspace'
+const WORKSPACE_ORGANIZATION = process.env.WORKSPACE_ORGANIZATION ?? ''
```

Comment:
`const WORKSPACE_ORGANIZATION = process.env.WORKSPACE_ORGANIZATION ?? 'Organization '` ?

Reply:
Not required I think.

Reply:
ok